src/App.js

```diff
@@ -1,16 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import ColorChoice from './components/ColorChoice';
+
 
 const App = () => {
+
+  // Update messages when like button is toggled
+  const [messages, setMessages] = useState(chatMessages);
```

Comment:
Nice setup for your main `messages` state variable and great work setting this at the very top of your component so it's easy to find!